data/factions/077_Tolan_Kingdom.lua

```diff
@@ -0,0 +1,27 @@
+-- Copyright © 2008-2021 Pioneer Developers. See AUTHORS.txt for details
+-- Licensed under the terms of the GPL v3. See licenses/GPL-3.txt
+
+local f = Faction:new('Tolan Kingdom')
+	:description_short('Tolan Kingdom')
+	:description('The Tolan Kingdom, famous (or infamous, depending on who you ask) for repelling Haber invasion forces 500 years ago.')
+	:homeworld(2,-9,-19,2,4) -- edge of Habers sphere of influence
+	:foundingDate(2337) -- as per timeline
+	:expansionRate(0) -- a single system only, but quite populous
+	:military_name('Tolan Defenceforce')
+	:police_name('Skymartial')
```

Comment:
I'd recommend 'Skymarshal' (or even better, 'Sky Marshal') in this case, as 'skymartial' is a noun-adjective pair being coerced into an adjective-adjective pair and doesn't mean anything.